src/hub/app.py

```diff
@@ -85,12 +84,20 @@ def create_app(config=None):
     app.add_api("swagger.yaml", pass_context_arg_name="request", strict_validation=True)
 
     app.app.hub_table = HubEvent(table_name=CFG.EVENT_TABLE, region=region, host=host)
+    app.app.subhub_deleted_users = SubHubDeletedAccount(
+        table_name=CFG.DELETED_USER_TABLE, region=region, host=host
+    )
 
     if not app.app.hub_table.model.exists():
         app.app.hub_table.model.create_table(
             read_capacity_units=1, write_capacity_units=1, wait=True
         )
 
+    if not app.app.subhub_deleted_users.model.exists():
+        app.app.subhub_deleted_users.model.create_table(
+            read_capacity_units=1, write_capacity_units=1, wait=True
```

Comment:
Not to do it now but we should revisit the read and write capacity as part of the database model review soon.

Reply:
Sounds good.  I think we should try to do that early next week.

Reply:
create Jira Story for it please.  Now before we forget.